modules/util/path.go

```diff
@@ -150,3 +153,23 @@ func StatDir(rootPath string, includeDir ...bool) ([]string, error) {
 	}
 	return statDir(rootPath, "", isIncludeDir, false, false)
 }
+
+// FileURLToPath extracts the path informations from a file://... url.
+func FileURLToPath(u *url.URL) (string, error) {
+	if u.Scheme != "file" {
+		return "", errors.New("URL scheme is not 'file': " + u.String())
+	}
+
+	path := u.Path
+
+	if runtime.GOOS != "windows" {
+		return path, nil
+	}
+
+	// If it looks like there's a Windows drive letter at the beginning, strip off the leading slash.
+	re := regexp.MustCompile("/[A-Za-z]:/")
```

Comment:
Move the statement as a global variable so that we don't need to compile every time.